src/store/actions/ActionTypes.ts

```diff
@@ -0,0 +1,5 @@
+const GET_MARKET = 'GET_MARKET';
+
+export {
```

Comment:
you could try this:
`export const GET_MARKET = 'GET_MARKET';`

and remove this part
export { GET_MARKET }